web/ui/website/pages/docs/index.vue

```diff
@@ -1,5 +1,54 @@
 <template>
-	<nuxt-content :document="pageData"></nuxt-content>
+	<div>
+		<nuxt-content v-if="currentRoute !== '/docs'" :document="pageData"></nuxt-content>
+		<div v-else>
```

Comment:
@Oluwadaminiola you actually don't need this hack. The content of this `div` should be the content of the page, then delete the `index.md` file.

`/docs` should always be the landing page, index will now be the `guide.md` which you've already created. @Oluwadaminiola